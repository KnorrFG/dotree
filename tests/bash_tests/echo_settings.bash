$DT -c echo_setting_on.dt y
$DT -c echo_setting_on.dt n
$DT -c echo_setting_off.dt y
$DT -c echo_setting_off.dt n
