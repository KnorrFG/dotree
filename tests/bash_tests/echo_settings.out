[?25l[?25hecho show me
show me
[?25l[?25hdon't show me
[?25l[?25hecho show me
show me
[?25l[?25hdon't show me
